### PR TITLE
Implements the Biesel Constitution

### PR DESCRIPTION
**Pull request by Ben10083**
Title

Used the litanybook code to make a book for Biesel consulars to show around and praise.

WIP until I can get enough lines into the .txt file.

- [x] Make a super awesome book
- [x] Steal the tcfl pamphlet sprite and change it up a bit
- [x] Fill out the .txt

### CODE REVIEW

**Comment by Ben10083** [comment]
!review

**Comment by Ben10083** [comment]
!wip

File: code/game/objects/items/devices/litanybook.dm [hunk 1]
```diff
@@ -42,4 +42,15 @@
 
 /obj/item/device/litanybook/gadpathur/Initialize()
 	. = ..()
-	randomquip = file2list("ingame_manuals/gadpathur.txt")
\ No newline at end of file
+	randomquip = file2list("ingame_manuals/gadpathur.txt")
+
+/obj/item/device/litanybook/biesel
```
Comment (NonQueueingMatt) [inline code/game/objects/items/devices/litanybook.dm:47]: If you are going to borrow litany book code, make the litany book mechanical functions into a generic class called `verse_book` or something, then make the litany book and this constitution into subtypes.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Review by JohnWildkins — APPROVED** [review]

File: code/game/objects/items/devices/versebook.dm [hunk 2]
```diff
@@ -13,11 +13,11 @@
 
 	var/list/randomquip = list()
 
-/obj/item/device/litanybook/Initialize()
+/obj/item/device/versebook/Initialize()
 	. = ..()
 	randomquip = file2list("ingame_manuals/dominia.txt")
```
Comment (NonQueueingMatt) [inline code/game/objects/items/devices/versebook.dm:18]: This should not be here. It's done below in the tribunal book.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Comment by Ben10083** [comment]
!review

File: code/game/objects/items/devices/versebook.dm [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+		return
+	reading = TRUE //begin reading
+
+	user.visible_message(SPAN_NOTICE("[user] begins to flip through [src]."))
+	playsound(loc, 'sound/bureaucracy/bookopen.ogg', 50, 1)
+
+	var/q // recycled from tip of the day code. it just works!(TM)
+	var/quip // in actuality it randomly determines a value, then uses that value to select a particular line from the txt below
+	if(quip)
+		q = quip
```
Comment (NonQueueingMatt) [inline code/game/objects/items/devices/versebook.dm:31]: This quip variable looks entirely useless...? Why is it there?

File: code/game/objects/items/devices/versebook.dm [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+	var/reading = FALSE
+
+	drop_sound = 'sound/items/drop/book.ogg'
+
+	pickup_sound = 'sound/items/pickup/book.ogg'
+
+	var/list/randomquip = list()
+
+/obj/item/device/versebook/attack_self(mob/user)
+	user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN)
```
Comment (NonQueueingMatt) [inline code/game/objects/items/devices/versebook.dm:18]: Stop the user from reading if randomquip is empty.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/game/objects/items/devices/versebook.dm [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+	var/list/randomquip = list()
+
+/obj/item/device/versebook/attack_self(mob/user)
+	if(!length(randomquip))
+		return
+	user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN)
+
+	if(reading) //So you can't read twice.
+		to_chat(usr, SPAN_WARNING("You are already reading this book."))
```
Comment (NonQueueingMatt) [inline code/game/objects/items/devices/versebook.dm:23]:
```suggestion
		to_chat(user, SPAN_WARNING("You are already reading this book."))
```

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**